x-pack/plugins/index_lifecycle_management/public/plugin.tsx

```diff
@@ -49,7 +49,7 @@ export class IndexLifecycleManagementPlugin
         id: PLUGIN.ID,
         title: PLUGIN.TITLE,
         order: 2,
-        mount: async ({ element, history, setBreadcrumbs }) => {
+        mount: async ({ element, history, setBreadcrumbs, managementPageLayout }) => {
```

Comment:
**Step 1:** Declare the injected `managementPageLayout` within the plugin's `start` lifecycle method. Note that we have to pass it into wherever context is being used, as seen below on line 80.